specificatie/domain.yaml

```diff
@@ -203,6 +203,15 @@ components:
                         De identificaties van de stukdelen (paragrafen in een akte met een rechtsfeit) waarin dit beslag is vermeld
           items:
             type: "string"
+    BijbehorendAppartementsrecht:
+      type: "object"
+      description: |
+                    De appartementsrechten die bij deze KadastraalOnroerende Zaak (grondperceel) horen
```

Comment:
Moet de decsription niet net als de naam van deze property in enkelvoud?
Dus:

`Het appartementsrecht dat bij deze KadastraalOnroerende Zaak (grondperceel) hoort`

Reply:
Ik zie nu dat dit niet het geval is. Het component is weliswaar enkelvoudig maar de property die gebruik maakt van dit component is meervoudig.